tokens/_aliases.yml

```diff
@@ -212,3 +212,5 @@ aliases:
   font-stack-firefox: "Metropolis, Inter, X-LocaleSpecific, sans-serif"
   font-stack-mono: "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace"
   font-stack-mozilla: "'Zilla Slab', Inter, X-LocaleSpecific, sans-serif"
```

Comment:
used in protocol here: https://github.com/mozilla/protocol/blob/main/assets/sass/protocol/includes/themes/_mozilla.scss#L12